src/main/java/invtweaks/InvTweaksConfigManager.java

```diff
@@ -251,5 +303,44 @@ private boolean extractFile(@NotNull ResourceLocation resource, @NotNull File de
             return false;
         }
     }
+    
+    private List<ResourceLocation> getResourceNames(String inPath) {
```

Comment:
This seems fairly awkward: It'd allow overriding the included ones with a resource pack (as is the case with the default tree, and why it's loaded that way), but there's no facility in place to mark new files matching from other resource packs.

IIRC the closest thing in vanilla is the language loader and I believe adding new ones requires checking a specific file in the pack that lists the properties of the new language.

Reply:
I suppose I should also point out that feature of loading defaults from resource packs was slightly ill-conceived and mildly pointless, because you tend to load the game without any resource packs set up initially, which means it loads the included defaults and then doesn't bother checking again unless the files are deleted.

Reply:
I honestly hadn't considered files coming from other resource packs.  I did it so I could get the seed files out of the jar and into place to be used.  What should I consider instead?

Reply:
It's been forever since I've looked at the resource stuff but I don't remember it being simple to enumerate there.

Since the specifics of *why* it's using ResourceLocation aren't really important (I don't think anyone's really using it, and it's awkward to set up a resource pack correctly to work, *and* since it all relies on extracted files it's unlikely to trigger) it's probably safe to leave as-is.

Like I said, it's a weird thing that sounded like a neat idea but in practice it's not really an advantage due to the implementation.

Reply:
I looked for an enumeration function and there (still) isn't one.